courses/management/commands/update_courses.py

```diff
@@ -156,7 +152,7 @@ def update_course(self, mongo_course, assign_universities=False):
                 self.stdout.write('\t No university assigned '
                 'to "{}"\n'.format(key))
         course.is_active = True
-        #course.show_in_catalog = bool(mongo_course.ispublic)
+        course.show_in_catalog = mongo_course.catalog_visibility.lower() == "both"
```

Comment:
Where does this 'both' value come from? Isn't it stored in a variable somewhere?